specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/UpdateRuns.json

```diff
@@ -0,0 +1,282 @@
+      "get": {
+        "x-ms-examples": {
+          "Get the list of update run.": {
+            "$ref": "./examples/UpdateRuns/List.json"
+          }
+        },
+        "tags": [
+          "UpdateRuns"
+        ],
+        "description": "Get the list of update runs.",
+        "operationId": "UpdateRuns_List",
+        "parameters": [
+          {
+            "$ref": "Update.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "UpdateLocations.json#/parameters/UpdateLocationParameter"
+          },
+          {
+            "$ref": "Updates.json#/parameters/UpdateParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/UpdateRunList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/System.{location}/providers/Microsoft.Update.Admin/updateLocations/{updateLocation}/updates/{update}/updateRuns/{runId}": {
+      "get": {
+        "x-ms-examples": {
+          "Get an update run.": {
+            "$ref": "./examples/UpdateRuns/Get.json"
+          }
+        },
+        "tags": [
+          "UpdateRuns"
+        ],
+        "description": "Get the list of update locations",
+        "operationId": "UpdateRuns_Get",
+        "parameters": [
+          {
+            "$ref": "Update.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "UpdateLocations.json#/parameters/UpdateLocationParameter"
+          },
+          {
+            "$ref": "Updates.json#/parameters/UpdateParameter"
+          },
+          {
+            "$ref": "#/parameters/UpdateRunParameter"
+          },
+          {
+            "$ref": "Update.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/UpdateRun"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/System.{location}/providers/Microsoft.Update.Admin/updateLocations/{updateLocation}/updates/{update}/updateRuns/{runId}/rerun": {
+      "post": {
+        "x-ms-examples": {
+          "Rerun an update.": {
+            "$ref": "./examples/UpdateRuns/Rerun.json"
+          }
+        },
+        "tags": [
+          "UpdateRuns"
+        ],
+        "description": "Get a specific update version.",
```

Comment:
Is this description accurate?

Reply:
The update team signed off on the spec after we asked them to review and make sure the descriptions are okay.  Guess they missed this.  I have changed it to something that is more correct.